aiida_fleur/tests/workflows/test_create_magnetic_workchain.py

```diff
@@ -0,0 +1,294 @@
+        assert outstruc != None
+
+        assert node.is_finished_ok
+        assert False
+        # check output
+        #distance, bulk modulus, optimal structure, opt scaling
+        #assert abs(outpara.get('scaling_gs') - 0.99268546558578) < 10**14
+        #assert outpara.get('warnings') == ['Groundstate volume was not in the scaling range.']
+        #assert outpara.get('info') == ['Consider rerunning around point 0.9926854655857787']
+
+    @pytest.mark.skip
+    @pytest.mark.timeout(500, method='thread')
+    def test_fleur_eos_validation_wrong_inputs(
+        self, run_with_cache, mock_code_factory, generate_structure2
+    ):
+        """
+        Test the validation behavior of FleurEosWorkChain if wrong input is provided it should throw
+        an exitcode and not start a Fleur run or crash
+        """
+        from aiida.orm import Dict
+
+        # prepare input nodes and dicts
+        options = {
+            'resources': {
+                "num_machines": 1,
+                "num_mpiprocs_per_machine": 1
+            },
+            'max_wallclock_seconds': 5 * 60,
+            'withmpi': False,
+            'custom_scheduler_commands': ''
+        }
+        options = Dict(dict=options).store()
+
+        FleurCode = mock_code_factory(
+            label='fleur',
+            data_dir_abspath=os.path.join(
+                os.path.dirname(os.path.abspath(__file__)), 'calc_data_dir/'
+            ),
+            entry_point=CALC_ENTRY_POINT,
+            ignore_files=['cdnc', 'out', 'FleurInputSchema.xsd', 'cdn.hdf', 'usage.json', 'cdn??']
+        )
+        InpgenCode = mock_code_factory(
+            label='inpgen',
+            data_dir_abspath=os.path.join(
+                os.path.dirname(os.path.abspath(__file__)), 'calc_data_dir/'
+            ),
+            entry_point=CALC2_ENTRY_POINT,
+            ignore_files=['_aiidasubmit.sh', 'FleurInputSchema.xsd']
+        )
+
+        wf_parameters = Dict(dict={'points': 9, 'step': 0.002, 'guess': 1.00, 'wrong_key': None})
+        wf_parameters.store()
+        structure = generate_structure2()
+        structure.store()
+
+        ################
+        # Create builders
+        # interface of exposed scf is tested elsewhere
+
+        # 1. create builder with wrong wf parameters
+        builder_additionalkeys = FleurEosWorkChain.get_builder()
+        builder_additionalkeys.structure = structure
+        builder_additionalkeys.wf_parameters = wf_parameters
+        builder_additionalkeys.scf.fleur = FleurCode
+        builder_additionalkeys.scf.inpgen = InpgenCode
+
+        ###################
+        # now run the builders all should fail early with exit codes
+
+        # 1. structure and fleurinp given
+        out, node = run_get_node(builder_additionalkeys)
+        assert out == {}
+        assert node.is_finished == True
+        assert node.is_finished_ok == False
+        assert node.exit_status == 230
```

Comment:
The test should be renamed despite it is not implemented to avoid the confusion.

Reply:
It is a minor problem which will be solved when the working tests will be implemented